packages/ckeditor5-list/src/listconfig.ts

```diff
@@ -171,14 +171,17 @@ export interface ListPropertiesStyleConfig {
 
 	/**
 	 * Defines which list styles should be available in the UI.
-	 * Accepts a configuration object with numbered and bulleted styles or `true` to enable all styles.
+	 * Accepts a configuration object with numbered and bulleted styles.
+	 *
+	 * **Note**: This configuration works only with
+	 * {@link module:list/listproperties~ListProperties list properties}.
```

Comment:
I'd move this below the code snippet, above the `@default` block, just to keep the consistency with other configuration options 🙂